django_admin_listfilter_dropdown/templatetags/django_admin_listfilter_dropdown_tags.py

```diff
@@ -0,0 +1,17 @@
+from django.urls import reverse
+from django import template
+
+
+register = template.Library()
+
+
+@register.simple_tag(takes_context=True)
+def get_autocomplete_url(context):
```

Comment:
looking for a way to reverse the correct admin:app_label_model_name_autocomplete name in the template so that the templatetag can be removed.